pages/series/index.vue

```diff
@@ -98,7 +98,6 @@ const parsedEventSeries = computed(() => {
 
 .page-event-series {
   position: relative;
-  background-color: var(--pale-blue);
```

Comment:
I am wondering why we removed this style. It seems to have caused a bug on the page 
<img width="1300" alt="Screenshot 2024-12-11 at 11 30 47 AM" src="https://github.com/user-attachments/assets/060353dc-f7f6-4ee5-87be-aba4b1c29348" />